boards/common/stm32/include/cfg_rtt_default.h

```diff
@@ -31,8 +31,21 @@ extern "C" {
  * On the STM32Lx platforms, we always utilize the LPTIM1.
  * @{
  */
-#define RTT_FREQUENCY       (1024U)             /* 32768 / 2^n */
-#define RTT_MAX_VALUE       (0x0000ffff)        /* 16-bit timer */
+#ifdef LPTIM1
+#ifdef CLOCK_LSE
+#define RTT_CLOCK_FREQUENCY (32768U)                    /* in Hz */
+#else
+#define RTT_CLOCK_FREQUENCY (CLOCK_LSI)                 /* in Hz */
+#endif
```

Comment:
Ups this doesn't work for boards defining `CLOCK_LSE` after loading `cfg_rtt_default.h`. I could fix it by:

- moving clock configuration to a `boards/common/stm32/include/x/cfg_clock_x_x_x.h` configuration file.
- including `cfg_rtt_default.h` after the clock definitions for those boards.
- Add `CLOCK_LSX` variable that would be set to `CLOCK_LSE` or `CLOCK_LSI` depending on the configuration, the precompiler could then resolve `#define RTT_CLOCK_FREQUENCY (CLOCL_LSX) `

Option two requires the smallest amount of changes.

Reply:
Simply removed this part for, `CLOCK_LSI` based `rtt` is not really supported currently